docs/development.md

```diff
@@ -147,16 +147,16 @@ A more convenient way to run checks is to via the scripts in the [check/](https:
 
 The above scripts are convenient and reasonably fast, but they often won't exactly match the results computed by the continuous integration builds run on travis.
```

Comment:
> they often won't exactly match the results computed by the continuous integration builds run on travis.

Sure, but `all` won't either since `all` is just running all the previous scripts so we don't have to run each one by one?

Reply:
I know you didn't touch these, but would be good to fix here.

Reply:
Updated entire section.